infra/core/managed-cluster/aks-cluster.bicep

```diff
@@ -60,6 +60,36 @@ var asoPlatformTeamMiRbacs = [
   }
 ]
 
+var systemNodePool = {
+  name: 'npsystem01'
+  mode: 'System'
+  count: cluster.npSystem.count
+  vmSize: 'Standard_DS2_v2'
+  osDiskSizeGB: cluster.npSystem.osDiskSizeGB
+  osDiskType: 'Ephemeral'
+  osType: 'Linux'
+  osSKU: 'Ubuntu'
+  minCount: cluster.npSystem.minCount
+  maxCount: cluster.npSystem.maxCount
+  vnetSubnetId: resourceId(vnet.resourceGroup, 'Microsoft.Network/virtualNetworks/subnets', vnet.name, vnet.subnetClusterNodes)
+  enableAutoScaling: true
+  enableCustomCATrust: false
+  enableFIPS: false
+  enableEncryptionAtHost: false
```

Comment:
prob should be true for encryption at host...

Reply:
Will pick it up in the next pr.